cli/src/command.rs

```diff
@@ -732,3 +738,17 @@ pub fn run() -> Result<()> {
 	}
 	Ok(())
 }
+
+/// Returns an error if a secure validator cannot be built for the target OS and architecture.
+fn can_run_as_secure_validator() -> std::result::Result<(), String> {
+	#[cfg(not(target_os = "linux"))]
+	let result = Err("Must be on Linux to run a validator securely.".into());
+
+	#[cfg(all(target_os = "linux", not(target_arch = "x86_64")))]
+	let result = Err("Must be on x86_64 to run a validator securely.".into());
```

Comment:
Maybe a little bit less jargon-y? Normal users might not necessarily know what `x86_64` is.

```suggestion
	#[cfg(not(target_os = "linux"))]
	let result = Err("Running a validator is only supported on Linux.".into());

	#[cfg(all(target_os = "linux", not(target_arch = "x86_64")))]
	let result = Err("Running a validator is only supported on CPUs from the x86_64 family (usually Intel or AMD).".into());
```